### ISSUE TITLE
Precondition from watchface - error

### ISSUE THREAD

**Issue by claire078** [itsMeDavidV/Watchla-Support#343]
I get this error messages when start precondition from watchface. 
![incoming-4AB2EBBB-A10F-4A23-811E-59A920033131](https://github.com/itsMeDavidV/Watchla-Support/assets/46406097/3c9529d0-daa8-4a52-9307-0ec585c97a40)
![incoming-0CCD3EFD-8961-40BB-80F0-9F63F1FA146D](https://github.com/itsMeDavidV/Watchla-Support/assets/46406097/a2ff4b41-6f7b-42d9-9a8c-f761e70966f3)
![incoming-9D29DA1C-5FE9-49F2-B331-279BC4F236B4](https://github.com/itsMeDavidV/Watchla-Support/assets/46406097/f5310b93-5916-4556-833d-81c7632c6b5e)

Car was sleeping, v3.7.1.
Apple watch ultra, watchos 10.4, does not matter which watch face.

**Comment by claire078** [comment]
3.7.2 still exist this problem.
![9184D318-CFB5-413F-8E87-15BA5D3530AA_4_5005_c](https://github.com/itsMeDavidV/Watchla-Support/assets/46406097/a9b56393-9be6-4d60-8669-a48e12026d1f)

**Comment by itsMeDavidV** [comment]
Hi @claire078, thanks for reporting. The 408 error appears because your vehicle is still waking and/or connecting to the internet. 408 errors when sending your commands are more prevalent now due to a Tesla API change but rest-assured were taking steps to adapt to this, so you should definitely stay tuned to new app updates.

The 401 error is a bit of different issue. Are you still seeing 401 errors?

**Comment by claire078** [comment]
> Hi @claire078, thanks for reporting. The 408 error appears because your vehicle is still waking and/or connecting to the internet. 408 errors when sending your commands are more prevalent now due to a Tesla API change but rest-assured were taking steps to adapt to this, so you should definitely stay tuned to new app updates.
> 
> The 401 error is a bit of different issue. Are you still seeing 401 errors?

Yes, it occurs sometimes, but I experienced with precondition from watch face.

**Comment by CassandraMarie** [comment]
Hi @claire078, we apologize for the delayed response, are you still experiencing issues with Watchla?
